.vib/mediawiki/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+file:
+  /opt/bitnami/mediawiki/cache:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/mediawiki/images:
+    filetype: symlink
+    linked-to: /bitnami/mediawiki/images
+    exists: true
+  /bitnami/mediawiki/LocalSettings.php:
+    mode: "0644"
+    filetype: file
+    exists: true
+# Checks the user running the container is non-root
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
```

Comment:
I think I will copy this and use it instead of my regex! Nice!

Reply:
Thanks! I decided to follow this approach instead of using a regex because I kept finding corner-cases and this seemed simpler. Any improvement to it is welcomed though.